### PR TITLE
Add org.kde.kmines

### PR DESCRIPTION
**Pull request by nicolasfella**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by nicolasfella** [comment]
bot, build org.kde.kmines

**Comment by nicolasfella** [comment]
bot, build org.kde.kmines

**Comment by nedrichards** [comment]
This reports about wanting the --device=dri permission for GPU acceleration, other than that lgtm.

File: org.kde.kmines.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "id": "org.kde.kmines",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kmines",
+    "rename-icon": "kmines",
+    "finish-args": ["--share=ipc", "--socket=fallback-x11", "--socket=wayland" ],
```
Comment (cho2) [inline org.kde.kmines.json:8]:
```
    "finish-args": [
        "--share=ipc",
        "--socket=fallback-x11",
        "--socket=wayland",
        "--device=dri"
    ],
```

**Comment by cho2** [comment]
cc @flathub/kde

**Comment by travier** [comment]
Not tested but LGTM with `dri` added as mentioned already.

**Comment by nicolasfella** [comment]
bot, build org.kde.kmines

**Comment by cho2** [comment]
cc @barthalion @nedrichards

**Pull request closed without merging**